allyourpasswords/Views/ToolbarCustomView.swift

```diff
@@ -9,45 +9,45 @@
 import Cocoa
 
 class ToolbarCustomView: NSView {
-
+    
```

Comment:
What's up with all of the whitespace in this file?

Reply:
Hmm. Not sure - sorry! I'll fix this up.